.github/renovate.json

```diff
@@ -76,5 +76,5 @@
       "groupSlug": "all-go-minor-patch"
     }
   ],
-  "reviewersFromCodeOwners": true
+  "reviewers": ["team:analysis-cloud-squad"]
```

Comment:
Now it should work according to the doc: https://github.com/renovatebot/renovate/blob/main/docs/usage/configuration-options.md#reviewers